src/elements/content-sidebar/BoxAISidebar.tsx

```diff
@@ -24,14 +27,22 @@ export interface BoxAISidebarProps {
     onExpandPressed: () => void;
 }
 
-function BoxAISidebar() {
+function BoxAISidebar({ onExpandPressed }: BoxAISidebarProps) {
```

Comment:
Could we change the name to `onExpandClicked`? Pressed doesn't sound natural to me 🤔Usually we name it as click

Reply:
I think it's fine to have click in it `onExpandClicked` it indicates that it's a click handler